I'm seeing another issue with the `MemoryLeakTest` when running the full test suite with `maven clean install`. 

````
Running com.dooapp.fxform.issues.MemoryLeakTest
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.106 sec <<< FAILURE!
testMemoryLeak(com.dooapp.fxform.issues.MemoryLeakTest)  Time elapsed: 0.106 sec  <<< FAILURE!
java.lang.AssertionError: expected null, but was:<FXForm@11fd4cd6>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotNull(Assert.java:756)
	at org.junit.Assert.assertNull(Assert.java:738)
	at org.junit.Assert.assertNull(Assert.java:748)
	at com.dooapp.fxform.issues.MemoryLeakTest.testMemoryLeak(MemoryLeakTest.java:93)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.dooapp.fxform.JavaFXRule$1.evaluate(JavaFXRule.java:45)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
````

This issue does not happen when running the test manually from IntellijIdea. I'm wondering whether it's related to `GcFinalization.awaitFullGc() ` from `com.antkorwin:common-utils` not performing as expected with Java 21. Since there is no more recent version for that is also tried to switch to `com.google.guava:guava-testlib:33.1.0-jre` using `GcFinalization.awaitFullGc()` but I get the same issue.

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -44,18 +44,6 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>javax.el</groupId>
-            <artifactId>el-api</artifactId>
-            <version>2.2</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>javax.xml.bind</groupId>
-            <artifactId>jaxb-api</artifactId>
-            <version>2.3.0</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
             <groupId>org.glassfish</groupId>
             <artifactId>jakarta.el</artifactId>
             <scope>test</scope>

--- a/core/src/test/java/com/dooapp/fxform/issues/Issue206Test.java
+++ b/core/src/test/java/com/dooapp/fxform/issues/Issue206Test.java
@@ -14,7 +14,6 @@ import javax.validation.constraints.Max;
 
 import static org.hamcrest.core.IsCollectionContaining.hasItems;
 import static org.junit.Assert.assertThat;
-//import static org.junit.internal.matchers.IsCollectionContaining.hasItems;
 
 public class Issue206Test {
 

--- a/core/src/test/java/com/dooapp/fxform/issues/Issue207Test.java
+++ b/core/src/test/java/com/dooapp/fxform/issues/Issue207Test.java
@@ -11,7 +11,6 @@ import org.junit.Rule;
 import org.junit.Test;
 
 import static org.junit.Assert.assertThat;
-//import static org.junit.internal.matchers.IsCollectionContaining.hasItems;
 import static org.hamcrest.core.IsCollectionContaining.hasItems;
 public class Issue207Test {
 

--- a/demo/pom.xml
+++ b/demo/pom.xml
@@ -38,11 +38,11 @@
             <artifactId>javafx-swing</artifactId>
             <version>${javafx.version}</version>
         </dependency>
-        <dependency>
-            <groupId>javax.xml.bind</groupId>
-            <artifactId>jaxb-api</artifactId>
-            <version>2.3.0</version>
-        </dependency>
+<!--        <dependency>-->
+<!--            <groupId>javax.xml.bind</groupId>-->
+<!--            <artifactId>jaxb-api</artifactId>-->
+<!--            <version>2.3.0</version>-->
+<!--        </dependency>-->
         <dependency>
             <groupId>org.hibernate.validator</groupId>
             <artifactId>hibernate-validator</artifactId>

--- a/demo/pom.xml
+++ b/demo/pom.xml
@@ -38,11 +38,6 @@
             <artifactId>javafx-swing</artifactId>
             <version>${javafx.version}</version>
         </dependency>
-<!--        <dependency>-->
-<!--            <groupId>javax.xml.bind</groupId>-->
-<!--            <artifactId>jaxb-api</artifactId>-->
-<!--            <version>2.3.0</version>-->
-<!--        </dependency>-->
         <dependency>
             <groupId>org.hibernate.validator</groupId>
             <artifactId>hibernate-validator</artifactId>

